docs/tutorials/goldenpath.md

```diff
@@ -0,0 +1,518 @@
+---
+id: goldenpath
+title:  Building Golden Path
+description: Tutorial that explains adding a command line handler, network variables (client and server-controlled), network transforms for auto-movement, and RPC introduction.
+---
+
+In this tutorial we will build on the work we have already done in Hello World and add in a few more features. As there are some minor differences between this tutorial and the Hello World series we will be starting afresh. You do not have to have completed either of the Hello World tutorials to complete this one, however, if you are new to Unity then we recommended that you  complete them in order to familiarise yourself with Unity. 
+
+We will be covering the following:
+
+- Adding a command line handler (to make launching easier)
+- Network variables (client and server-controlled) 
+- Network transforms
+- RPCs
+
+:::funfact
+
+In the context of software or information modelling, a Golden Path (sometimes called happy path) is a default scenario featuring no exceptional or error conditions.
+:::
+
+## Requirements
+
+This tutorial requires an MLAPI-supported version of Unity (2019.4+).
+
+## Create a new project in Unity
+
+1. Open the Unity Hub.
+1. Click New. 
+1. Select type ‘3D’
+1. Rename the project **GoldenPath**.
+1. Select the location to save the project.
+
+:::note
+Make note of this path. You will need it  later in the [tutorial](#creating-a-command-line-helper), when you are testing building the project from the command line. For this guide we used a MAC and our path was `~/dev/mlapi-golden-path/` on a MAC your path may be slightly different.
```

Comment:
Do you mean it may differ on a Windows machine or just on their machine? Also MAC should be Mac. MacOS or OSX are names for the operating system.
```suggestion
Make note of this path. You will need it  later in the [tutorial](#creating-a-command-line-helper), when you are testing building the project from the command line. For this guide, we used a Mac and our path was `~/dev/mlapi-golden-path/`. On a Windows machine, your path may be slightly different.
```

Reply:
Done